ada/testsuite/tests/properties/find_all_references/test.out

```diff
@@ -1,29 +1,36 @@
 All references to A from a.ads:
```

Comment:
You don't seem to test find_all_refs on a built-in operator right ?

Reply:
U fuh'in w0t m8??